### PR TITLE
SEO optimization for "what-is-observability"

### PR DESCRIPTION
**Pull request by mmanciop**
## Done

- Clean up the META, add ALT with the relevant keywords based on suggestions by [Moz](https://analytics.moz.com/pro/research/page-grader?keyword=what%20is%20observability&url=https%3A%2F%2Fubuntu.com%2Fobservability%2Fwhat-is-observability)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes the lack of Rank #1 on the [SEO Dashboard](https://datastudio.google.com/u/0/reporting/07ad6bee-cd8b-47b1-ad41-978aacccdfe0/page/rPmpB?mkt_tok=MDY2LUVPVi0zMzUAAAF8USKndkHzSYYjJAJze2BYJniiG8arGmrGxG2ah6yg_s2k-wzd9XsYKQ0RYmHAyRBOm7Oo9diZQ-k7Q5k1_ZqKL7nUi_e3IUWelI4r6MNNB67k_w) (hopefully)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/observability/what-is-observability.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% block title %}What is observability{% endblock %}
 
-{% block meta_description %}Observability is fundamental for applications and systems. Learn how it relates with monitoring and the different types of telemetry that is collected and analyzed in the state of the art of open source observability.{% endblock %}
+{% block meta_description %}What is observability, the difference between observability and monitoring (observability vs monitoring) and how the different types of telemetry that are collected and analyzed in the state of the art of open-source observability.{% endblock %}
```
Comment (simskij) [inline templates/observability/what-is-observability.html:5]:
```suggestion
{% block meta_description %}What is observability, the difference between observability and monitoring (observability vs monitoring), and how are the different types of telemetry collected and analyzed in the state of the art of open-source observability.{% endblock %}
```

**Review by simskij — CHANGES_REQUESTED** [review]

File: templates/observability/what-is-observability.html [hunk 2]
```diff
@@ -173,7 +172,7 @@ <h3 class="p-heading--4">Making the most out of your metrics, logs, distributed
             <p>
                 {{ image (
                     url="https://assets.ubuntu.com/v1/cf7d3849-telemetry-map-v2.svg",
-                    alt="",
+                    alt="Logs, metrics, distributed traces, production profiling, synthetic monitoring and end-user monitoring all contribute to observability, documenting different aspects of the performance of a system",
```
Comment (anthonydillon) [inline templates/observability/what-is-observability.html:175]: Image alt text are not a way to include keywords but should describe the image to someone using a screen reader. Do you think this effectively describes that image to that audience?

Comment (mmanciop) [reply templates/observability/what-is-observability.html:175]: I think it describes it a bit. Could be of course longer: that image is making _a lot_ of work. What is an acceptable length for an `alt` tag?

@caldav what's your take on this matter?

Comment (caldav) [reply templates/observability/what-is-observability.html:175]: I would reword the following way to make it more descriptive: "Diagram of observability components of a system: logs, metrics, distributed traces, production profiling, synthetic and end-user monitoring". If you were using a screen reader, you'd want the big concept first, then the details. The rough limit is 15 words (~130 characters).

Comment (mmanciop) [reply templates/observability/what-is-observability.html]: I kept a little more info than that, there is no write-up yet in the page about the correlation among different types of telemetry.

**Review by sowasred2012 — APPROVED** [review]
LGTM
